[domaincontrol.com] HTTPSConnectionPool(host='domaincontrol.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f8acdc84410>: Failed to establish a new connection: [Errno 111] Connection refused'))
[macromedia.com] HTTPSConnectionPool(host='macromedia.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8acdc84050>, 'Connection to macromedia.com timed out. (connect timeout=None)'))
[msedge.net] HTTPSConnectionPool(host='msedge.net', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8acdc87110>, 'Connection to msedge.net timed out. (connect timeout=None)'))
[shifen.com] HTTPSConnectionPool(host='shifen.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f8acc9e7d90>, 'Connection to shifen.com timed out. (connect timeout=None)'))
